x-pack/legacy/plugins/advanced_ui_actions/public/np_ready/public/custom_time_range_action.test.ts

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// @ts-ignore
+import { findTestSubject } from '@elastic/eui/lib/test';
+import { skip } from 'rxjs/operators';
+import * as Rx from 'rxjs';
+import { mount } from 'enzyme';
+
+import { EmbeddableFactory } from '../../../../../../../src/legacy/core_plugins/embeddable_api/public/np_ready/public';
+import { TimeRangeEmbeddable, TimeRangeContainer, TIME_RANGE_EMBEDDABLE } from './test_helpers';
+import { TimeRangeEmbeddableFactory } from './test_helpers/time_range_embeddable_factory';
+import { CustomTimeRangeAction } from './custom_time_range_action';
+import { coreMock } from '../../../../../../../src/core/public/mocks';
+
+jest.mock('ui/new_platform');
```

Comment:
Do you need this? The idea for `jest.mock('ui/new_platform');` is to use it only in old tests, not in newly written ones. New tests can be written such that this line is not needed. Also, not in `np_ready` folder.

Reply:
apparently so :(  Without it I get:

```
  ● Test suite failed to run

    TypeError: Cannot read property 'chrome' of null

      20 | import { npStart } from '../new_platform';
      21 | 
    > 22 | export const recentlyAccessed = npStart.core.chrome.recentlyAccessed;
         |                                              ^
      23 | 

      at Object.chrome (../src/legacy/ui/public/persisted_log/recently_accessed.ts:22:46)
      at Object.<anonymous> (../src/legacy/ui/public/persisted_log/index.ts:24:1)
      at Object.<anonymous> (../src/legacy/ui/public/timefilter/time_history.js:21:1)
      at Object.<anonymous> (../src/legacy/core_plugins/data/public/query/query_bar/components/query_bar.tsx:27:1)
      at Object.<anonymous> (../src/legacy/core_plugins/data/public/query/query_bar/components/index.ts:20:1)
      at Object.<anonymous> (../src/legacy/core_plugins/data/public/query/query_bar/index.ts:20:1)
      at Object.<anonymous> (../src/legacy/core_plugins/data/public/query/query_service.ts:20:1)
      at Object.<anonymous> (../src/legacy/core_plugins/data/public/query/index.ts:20:1)
      at Object.<anonymous> (../src/legacy/core_plugins/data/public/plugin.ts:23:1)
      at Object.<anonymous> (../src/legacy/core_plugins/data/public/index.ts:21:1)
      at Object.<anonymous> (../src/legacy/core_plugins/kibana/public/discover/embeddable/search_embeddable.ts:33:1)
      at Object.<anonymous> (legacy/plugins/advanced_ui_actions/public/np_ready/public/custom_time_range_action.tsx:10:1)
      at Object.<anonymous> (legacy/plugins/advanced_ui_actions/public/np_ready/public/custom_time_range_action.test.ts:16:1)
```

The data plugin ends up `importing npStart.core.chrome`